corpora-api.yml

```diff
@@ -493,6 +497,131 @@ components:
     visibility:
       type: string
       enum: [public,private]
+    project:
+      type: object
+        properties:
+          description:
+            type: string
+          name:
+            type: string
+          id:
+            $ref: '#/components/schemas/project_uuid'
+          s3_bucket_key:
+            type: string
+          submitter:
```

Comment:
I've named this `owner` in the ORM to use broader terminology and imply access controls. Does this make sense to you @maniarathi @Bento007 ? Update if so, will update ORM if not.

Reply:
Hrrrmmm.. `submitter` personally makes more sense to me, but I could be convinced otherwise. What was the rational to making it `owner`?

Reply:
I see. `owner` makes sense to me as an internal name since it's a more common term in describing roles/access controls in software, while `submitter` does make more sense contextually/as an external name. The rationale, to be clear, was `owner` implies:
- this is a User in the the system
- this User has all the access permissions
- this User likely submitted the project

So actually, I'm fine either way considering `submitter`s contextual connotation.

Reply:
Is it possible for the submitter to not be the owner? If yes, then we should make the distinction. Conceptually it makes sense that the owner could change to someone other than the original submitter. I think we should keep submitter, until there is a need to also track a separate ownership trait. For now the submitter is the only owner.

Reply:
It's unclear whether in the future, there will be a difference between someone who "owns" a  project versus someone who "submits" a project. @calvinnhieu you make some good points that owner currently refers to a User who has the correct permissions in order to submit a project so I'm leaning towards changing it to `owner` now.

Reply:
Owner it is.